packages/cas/src/tests/__fixtures__/index.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * The code in this file originated from
+ * @see https://github.com/decentralized-identity/sidetree
+ * For the list of changes that was made to the original code
+ * @see https://github.com/transmute-industries/sidetree.js/blob/main/reference-implementation-changes.md
+ *
+ * Copyright 2020 - Transmute Industries Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const testBuffer = Buffer.from(
+  '1f8b08000000000000034d91db6ee2301086dfc5d73dc5b4bb2d7729a5a1453408e8525a21e4d893c4cdc1966343bc28efbe362c6d255f8cfff9e7f3cc788f18949a34a8ffb1474632a26120aa8aeb0a6a8dfa68c8c3cdb9685e2c4e696ca2e8f760160f9377f5ccec34bca6365946b3ccacdab45d1457193a4392689ac39147a8e6a2761405b224145c9a096a8ee83d922629391d833dba3973ce866735af33273ab3b6127c0f9435640e54e29b5f45f007144f39251eed6cf82ab8f3cf9e58cfbbc2b30b6d7de5c0a5a8da7af0a9de29adbb7f62dc9b3d9a84f5a2e5e805bfc5f1b99e878b30d4e5fd2bbef91b4c8a16f7623adebcba0a0f63f3227f8b69c03f37ed232b96228f52939adbebc9f41ea623bb9a8c47692b57b1a1a8f31d29299ac32610313a7743ffefdaf148d380f2f104742e98ef924892f0926bfb546fc597f15b7e8012b2935c800d33057058e5ba5b9fb9f9d49653f8b9caa3f2c402dcfb5e66232a58f8f0ab64583329f8e1b773ad65d3bfbcdced76174695175454a85b77eeacbb7fdffa6cbc2a020000',
+  'hex'
+);
+export const testBufferHash = 'EiDq2c7TX1y0TeI1393yt9x5Gk-teF--ttExfcCWvaAIWA';
+
+export const testString = 'anyCoreIndexFileBuffer';
```

Comment:
same for this.

Reply:
The test string was taken from the batch writer spec test fro sidechain.

```[[32m✓ should return without writing anything if operation queue is empty.^[[39m (0.004 sec)

616e79436f7265496e64657846696c65427566666572
anyCoreIndexFileBuffer
Get
Writing Buffer:  616e79436f7265496e64657846696c65427566666572
Buffer Content(ascii):  anyCoreIndexFileBuffer
Encoded Hash:  EiBIGF9XeghTuWcaBQOWpMgz4KQgremvgzfZ3t4MkF793g
```